specification/storage/resource-manager/Microsoft.Storage/stable/2017-06-01/examples/OperationsList.json

```diff
@@ -11,134 +11,119 @@
             "display": {
               "provider": "Microsoft Storage",
               "resource": "Storage Accounts",
-              "operation": "Create/Update Storage Account",
-              "description": "Creates a storage account with the specified parameters or update the properties or tags or adds custom domain for the specified storage account."
```

Comment:
How/why is this removing this metadata?

Reply:
To fix some model validation errors. I can remove the separate commit and submit it separately if preferred. There is a bug in VS Code with the schemas suggesting that authors add "descriptions" in example files, but the "description" isn't part of the Swagger properties